pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -60,6 +61,9 @@ func RunClusterInfo(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command) error
 		return err
 	}
 	printService(out, "Kubernetes master", client.Host)
+	if len(client.AlternateHosts) > 0 {
```

Comment:
Sorry, i can't figure out what i suppose to change. The only related option in kubectl is:
`-s, --server='': The address and port of the Kubernetes API server` and it doesnt affect `kubectl config view` in any way. Beside that this is how cluster part of my config looks like:

```
clusters:
- cluster:
    certificate-authority-data: REDACTED
    server: https://10.245.1.2
    servers:
    - https://10.245.1.3
    - https://10.245.1.4
  name: vagrant```